ivtest/vvp_reg.py

```diff
@@ -0,0 +1,93 @@
+#! python
+'''
+Usage:
+    vvp_reg
+    vvp_reg <list-paths>...
+
+<list-paths> is a list of files in the current working directory that
+            each contain a list of tests. By convention, the file has the
+            suffix ".list". The files will be processed in order, so tests
+            can be overridden if listed twice. If no files are given, a
+            default list is used.
+'''
+
+from docopt import docopt
+import test_lists
+import run_ivl
+
+
+def process_test(item: list) -> str:
+    '''Process a single test'''
+
+    # This is th name of the test, and the name of the main sorce file
+    it_key = item[0]
+    # This is the name of the subdirectory that contains the test.
+    it_directory = item[2]
+    # The type is broken into a test type and a list of compiler flags.
+    it_type = item[1].split(',')
+    it_args = it_type[1:len(it_type)]
```

Comment:
```suggestion
    it_args = it_type[1:]
```

Reply:
> But didn't want to offend anybody by suggesting perl is outdated, that's why I kept it for myself 🙂
I gave up my perl wizards cap a long time ago. I think most of us have switches to Python for our more serious programming tasks.

Reply:
> I think most of us have switches to Python for our more serious programming tasks.

Not me. I still use Perl regularly and dislike Python. But if somebody else is going to maintain it, that's fine by me :-)

Reply:
I did say most and not all on purpose :-)

Reply:
Oops, hope I didn't step on any feet here.